### PR TITLE
make into npm module

### PR DESCRIPTION
**Pull request by lilianammmatos**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jquery-operator",
+  "version": "1.0.0",
```
Comment (hellatan) [inline package.json]: really? this file wasn't there?

**Comment by lilianammmatos** [comment]
@hellatan got it to `npm install` on my local. :+1:

**Comment by lilianammmatos** [comment]
remove `jquery` dependency in `package.json`

**Comment by lilianammmatos** [comment]
*removed

**Pull request merged**